pkgs/by-name/ti/tiny-dfr/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, rustPlatform, fetchFromGitHub, pkg-config
+, cairo, gdk-pixbuf, glib, libinput, libxml2, pango, udev
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tiny-dfr";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "WhatAmISupposedToPutHere";
+    repo = "tiny-dfr";
+    rev = "v${version}";
+    hash = "sha256-oawKYrfXAQ5RFMdUCG7F12wHcnFif++44s2KsX9ns6U=";
+  };
+
+  cargoHash = "sha256-QOkztErJLFXPxCb8MvaXi7jGXeI5A0q8LwZtYddzUZE=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ cairo gdk-pixbuf glib libinput libxml2 pango udev ];
+
+  postConfigure = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
Why? I think this makes more semantic sense in configurePhase, since that's where build systems (when they actually support it like autoconf, cmake or meson, and unlike cargo) will do this sort of substitution based on configuration.

Reply:
Well, it's kinda a patch, so it should live in patchPhase

Reply:
is this blocking an addition of this pkg? I would personally go with patch phase since there is no real support for configuration upstream, however I don't think it's really essential

Reply:
There's no real support for patching upstream either, but the fact is that we have a configure phase, and replacing placeholder paths with environment-specific ones is exactly what the configure phase is for.  I don't intend to change it.